tests/contrib/google_generativeai/conftest.py

```diff
@@ -36,7 +36,6 @@ def mock_tracer(ddtrace_global_config, genai):
         pin = Pin.get_from(genai)
         mock_tracer = DummyTracer(writer=DummyWriter(trace_flush_enabled=False))
         pin._override(genai, tracer=mock_tracer)
-        pin.tracer._configure()
```

Comment:
This basically does nothing (it just restarts the agentwriter and could introduce flakiness), I removed it.